server/webhook.go

```diff
@@ -260,6 +260,268 @@ func (s *Server) registerWebhookRoutes(g *echo.Group) {
 		}
 		return c.String(http.StatusOK, strings.Join(createdMessageList, "\n"))
 	})
+	g.POST("/github/:id", func(c echo.Context) error {
```

Comment:
Consider to move some code to helper functions shared between github and gitlab.

Reply:
Addressed in https://github.com/bytebase/bytebase/pull/1974/commits/3d42e48088fd90940453663bf042af0c3fe38b83, it is now recommended to use "Split diff view" for review.